packages/target-decisioning-engine/src/index.js

```diff
@@ -21,6 +21,7 @@ export default async function TargetDecisioningEngine(config) {
   });
 
   let artifact = artifactProvider.getArtifact();
+  let geoContext = artifactProvider.getGeoContext();
```

Comment:
Looks like this isn't being used.  But it should be passed in down below in the `createDecisioningContext` method call.

Reply:
I thought it will be set on `context` object which passed to `createDecisioningContext` via `deliveryRequest`. Now explicitly passing in `geoContext` to `createDecisioningContext`.